gobblin-config-management/gobblin-config-client/build.gradle

```diff
@@ -18,6 +18,8 @@ dependencies {
     compile externalDependency.slf4j
     compile externalDependency.log4j
     compile externalDependency.typesafeConfig
+    compile externalDependency.azkaban
```

Comment:
If the integration tests bring a dependency on azkaban, they should be in a separate directory with its own build.gradle. The config client should not depend on azkaban.